broadcast/broadcast.go

```diff
@@ -0,0 +1,43 @@
+package broadcast
+
+import (
+	"context"
+)
+
+type BroadcastFactory func() Broadcaster
+
+type Broadcaster interface {
+	BestQuoter
+	// FastestQuoter
+	// FeeQuoter
+	// PolicyQuoter
+	// TransactionQuerier
+	// TransactionSubmitter
+	// TransactionsSubmitter
+}
+
+type compositeBroadcaster struct {
+	broadcasters []Broadcaster
+	strategy     Strategy
+}
+
+func NewCompositeBroadcaster(strategy Strategy, factories ...BroadcastFactory) Broadcaster {
+	var broadcasters []Broadcaster
+	for _, factory := range factories {
+		broadcasters = append(broadcasters, factory())
+	}
+	return &compositeBroadcaster{
+		broadcasters: broadcasters,
+		strategy:     strategy,
+	}
+}
+
+func (c *compositeBroadcaster) BestQuote(ctx context.Context, feeCategory, feeType string) error {
```

Comment:
### Something to consider:

Maybe we should always return something like Result struct, that apart from the Response from Arc will also contain status of the call: I mean something containing informations like:
Arc 1 - call fail with error
Arc 2 - call finished with success  + using response as a result
Arc 3 - skipped
Overall - success

Reply:
Ok, we can go this way - this response is mocked but I will think about it when implementing.